library/std/src/os/unix/process.rs

```diff
@@ -24,26 +24,29 @@ pub trait CommandExt: Sealed {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn uid(
         &mut self,
-        #[cfg(not(any(target_os = "vxworks", target_os = "espidf")))] id: u32,
-        #[cfg(any(target_os = "vxworks", target_os = "espidf"))] id: u16,
+        #[cfg(not(any(target_os = "vxworks", target_os = "espidf", target_os = "horizon")))]
```

Comment:
Nit: I feel like a private typedef would be warranted for this.

Reply:
Would that change be allowed, since this is a stable interface?

Reply:
To the best of my knowledge if you have a private typedef, then it is entirely interchangeable with the type being aliased, but asking on the `t-libs` on Zulip to make sure wouldn't hurt.

Reply:
I opened a thread here:
https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/Replacing.20std.20function.20arg.20type.20with.20private.20type.20def.3F

Reply:
Got an affirmative response, updated.